### ISSUE TITLE
Journal Quality Research Paper due June 1

### ISSUE THREAD

**Issue by patelharsh94** [chicagoedt/iarc_software#15]
We need to write a research paper on our project; which is due on June 1.

**Comment by lnsh4992** [comment]
Sounds like a lot of work lol.

**Comment by patelharsh94** [comment]
Lol I started on working on breaking this project up this morning and I
realized they weren't kidding when they said its near impossible.  I'll
post more stuff as I think of it.

**Comment by lnsh4992** [comment]
Yeah, I'm not too optimistic myself. But worth a shot, The visual part is
probably gonna be the hardest. I'll be back in Chicago on 9th. Should be
able to do a lot then. Hope nothing is out of our grasp lol.

**Comment by patelharsh94** [comment]
We can do it, I just know I personally haven't dedicated enough time; I
need to pick up some slack, and start working.  Also not experienced in
these things its my first project like this.  Hopefully we will pull
through.

**Comment by lnsh4992** [comment]
Feel the same way on both of those aspects. We just need to dedicate more
time and hopefully we'll be done with most of it by the end of the semester.
